index.html

```diff
@@ -70,4 +70,166 @@ <h3 class="supporting-text">I’am a Software Developer</h3>
+                            </ul>
+                    </div>
+                            <button class="card-btn">See Project</button>
+                </div>    
+
+                <div class="card item3">
+                    <div class="text-block">
+                        <h3 class="new">Profesional Art<br>Printing Data</h3>
+                        <p class="card-description">A daily selection of privately 
+                            personalized reads; no accounts or 
+                            sign-ups required. has been the industry's standard</p>
+                            <ul class="lang">
+                                <li><a href="@" class="btn btn-car">html</a></li>
+                                <li><a href="@" class="btn btn-car">bootstrap</a></li>
+                                <li><a href="@" class="btn btn-car">Ruby</a></li>
+                            </ul>
+                    </div>
+                            <button class="card-btn">See Project</button>
+                </div>
+
+                <div class="card item4">
+                    <div class="text-block">
+                        <h3 class="new">Profesional Art<br>Printing Data</h3>
+                        <p class="card-description">A daily selection of privately 
+                            personalized reads; no accounts or 
+                            sign-ups required. has been the industry's standard</p>
+                            <ul class="lang">
+                                <li><a href="@" class="btn btn-car">html</a></li>
+                                <li><a href="@" class="btn btn-car">bootstrap</a></li>
+                                <li><a href="@" class="btn btn-car">Ruby</a></li>
+                            </ul>
+                    </div>
+                            <button class="card-btn">See Project</button>
+                </div>
+
+                <div class="card item5">
+                    <div class="text-block">
+                        <h3 class="new">Profesional Art<br>Printing Data</h3>
+                        <p class="card-description">A daily selection of privately 
+                            personalized reads; no accounts or 
+                            sign-ups required. has been the industry's standard</p>
+                            <ul class="lang">
+                                <li><a href="@" class="btn btn-car">html</a></li>
+                                <li><a href="@" class="btn btn-car">bootstrap</a></li>
+                                <li><a href="@" class="btn btn-car">Ruby</a></li>
+                            </ul>
+                    </div>
+                            <button class="card-btn">See Project</button>
+                </div>
+
+                <div class="card item6">
+                    <div class="text-block">
+                        <h3 class="new">Profesional Art<br>Printing Data</h3>
```

Comment:
Well done on implementing this portfolio design 💪🏻. I have noticed that the project title - `<h3>`'s font-family does not match that of the design. And I think it would be nice that you find the correct font-family from the design and change it to that❗ Having a consistent font for all headings adds to a professional portfolio.

You can compare them below
| Your design | Figma design |
| --- | --- |
| <img width="283" alt="123" src="https://user-images.githubusercontent.com/4247740/144012222-3f831094-ff62-4f42-95a3-6f29151189f5.PNG"> | <img width="283" alt="1232" src="https://user-images.githubusercontent.com/4247740/144012261-b28d34a4-36d3-4dda-a76e-f7a77f1a1805.PNG"> |